test/Microsoft.ApplicationInsights.AspNet.Tests/FakeTelemetryChannel.cs

```diff
@@ -6,6 +6,8 @@
     // TODO: Remove FakeTelemetryChannel when we can use a dynamic test isolation framework, like NSubstitute or Moq
```

Comment:
Hopefully we can start using a dynamic test isolation framework like Moq or NSubstitute and stop maintaining the fakes in tests.